### PR TITLE
Publish AARs to Maven Central

### PR DESCRIPTION
**Pull request by mhqz**
Hi @ivilata!,

This PR adds support for publishing Ouinet's AARs to Maven Central using `gradle-nexus.publish-plugin` and a GPG sub-key.

It also upgrades Gradle to use a version of the signing methods that supports specifying a `keyID`.

Please let me know if you have any questions or comments :)

### CODE REVIEW

File: android/ouinet/build.gradle [hunk 1]
```diff
@@ -63,3 +63,11 @@ dependencies {
     testImplementation 'com.android.support.test:runner:1.0.2'
     testImplementation 'org.assertj:assertj-core:3.11.1'
 }
+
+ext {
+    PUBLISH_GROUP_ID = 'ie.equalit.ouinet'
+    PUBLISH_VERSION = '1.5.0'
```
Comment (ivilata) [inline android/ouinet/build.gradle:69]: Just a reminder that this version may be extracted from `version.txt` in the main source directory.

Comment (mhqz) [reply android/ouinet/build.gradle]: Hi @ivilata, the change is implemented now in 4b3c78f0

File: android/build-scripts/publish-module.gradle [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            release(MavenPublication) {
+                // The coordinates of the library, being set from variables that
+                // we'll set up later
+                groupId PUBLISH_GROUP_ID
+                artifactId PUBLISH_ARTIFACT_ID
+                version PUBLISH_VERSION
+
+                // Two artifacts, the `aar` (or `jar`) and the sources
+                artifact androidSourcesJar
+                artifact("$buildDir/outputs/aar/ouinet-release.aar")
```
Comment (mhqz) [inline android/build-scripts/publish-module.gradle:30]: Hi @ivilata, I just realized that I'd probably need to change this line to support publishing artifacts from different architectures in the same package. I'll check it tomorrow and will push the changes to the same PR.

Comment (mhqz) [reply android/build-scripts/publish-module.gradle:30]: Fixed in c316711. Now the artifacts will be published in Maven Central with the following structure: `ouinet-arm64-v8a:0.19.0`

**Comment by ivilata** [comment]
Great work @mhqz! I checked that it doesn't break the build of CENO Browser either.

Merging in! :smile:

**Pull request merged**